### DISCUSSION TITLE
How can I display the contents of a cell when I hover over a row in the displayed table?

### DISCUSSION THREAD

**Discussion by Zapzatron** [dbeaver/dbeaver#20326] [Q&A] [answered]
How can I display the contents of a cell when I hover over a row in the displayed table?
I hover over the row, the text appears but disappears after a few seconds.
Also, not all of the cell content is shown

**Comment by E1izabeth** [comment, marked as answer]
You can click one the cell and see its content in the [Value panel](https://www.youtube.com/watch?v=hylFIEF1W-Q).

**Reply by Zapzatron** [reply]
Thank you so much, this makes it so much easier to use
